rts/Game/Camera/SpringController.cpp

```diff
@@ -73,6 +75,7 @@ void CSpringController::ConfigUpdate()
 	RECOIL_DETAILED_TRACY_ZONE;
 	scrollSpeed = configHandler->GetFloat("CamSpringScrollSpeed") * 0.1f;
 	fov = configHandler->GetFloat("CamSpringFOV");
+	minDist = configHandler->GetFloat("CamSpringMinHeight");
```

Comment:
I think this approach will not work consistently and will get worse when camera angle is flatter. `pos` in spring controller is a focus point on the ground, `dist` is a distance from that point using current rotation (`dir`), actual camera position is calculated in method `GetPos()`. There's some code that limits/manipulates camera height in method `SmoothCamHeight()`  by changing point of focus (`pos`). Maybe it can be reused?

Reply:
What overhead camera calls height is actually zoom distance, and I carried over the name here to spring camera. I will rename both to zoom distance.

Reply:
I feel their concern is mostly about the experience of the limit than the name itself. That's the point I raised in my note, is the dist checked against minDist the ray traced from frustum position into dir or the actual height to ground?

Reply:
e.g. your actual height might be below mindist but looking at the horizon the check might not work

Reply:
All of these "heights" are rays out of the camera frustum, therefore I will rename the config values to make it clear.